README.md

```diff
@@ -54,6 +54,90 @@ Block block = ckbService.getBlock(blockHash).send().getBlock();
 
 You can see more JSON-RPC requests from [RPC Document](https://github.com/nervosnetwork/ckb/blob/develop/rpc/README.md)
 
+#### Transfer
+
+`console/TransactionExample.java` provides `sendCapacityWithSinglePrivateKey` method with inputs which belong to single private key 
+and `sendCapacityWithMultiPrivateKey` method with inputs which belong to multi private keys.
+
+You can reference detail example in `console/TransactionExample.java`.
+
+```Java
+  private static final String NODE_URL = "http://localhost:8114";
+  private static final BigInteger UnitCKB = new BigInteger("100000000");
+  private static CKBService ckbService;
+  private static List<Account> Accounts;
+
+  static {
+    HttpService.setDebug(false);
+    ckbService = CKBService.build(new HttpService(NODE_URL));
+    Accounts =
+        Arrays.asList(
+            new Account(
+                "08730a367dfabcadb805d69e0e613558d5160eb8bab9d6e326980c2c46a05db2",
+                "ckt1qyqxgp7za7dajm5wzjkye52asc8fxvvqy9eqlhp82g"),
+            new Account(
+                "a202386cb9e46cecff9bc14b748b714c713075dd964c2507c8a8900540164959",
+                "ckt1qyqtnz38fht9nvmrfdeunrhdtp29n0gagkps4duhek"),
+            new Account(
+                "89b773ec5cf97b8fd2cf280ab1e37cd658dc28d84bac8f8dda4a8646cc08d266",
+                "ckt1qyqxvnycu7tdtyuejn3mmcnl4y09muxz8c3s2ewjd4"));
+  }
+
+  public static void main(String[] args) throws Exception {
+    String minerPrivateKey = "e79f3207ea4980b7fed79956d5934249ceac4751a4fae01a0f7c4a96884bc4e3";
+    List<Receiver> funds =
```

Comment:
why the variable name is `funds` but it type is Receiver?

Reply:
funds mean that they have more CKB. And they will send capcity to other accounts, so it is just a account name.

Reply:
ok

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/170/commits/fd1a27e6e9b62241b184467523cb9384b5d5c0f6